helm/a3s/values.yaml

```diff
@@ -0,0 +1,57 @@
+replicas: 1
+updateStrategy: RollingUpdate
+clusterIP: None
+
+log:
+    level: info
+    format: console
+
+global:
+  audience: 
+  issuer: 
+  imageRegistry: gcr.io/aporetodev
+  imageTag: latest
+  terminationGracePeriodSeconds: 60
+  log:
```

Comment:
the default value should be left empty to fallback to a3s flags defaults. possible values should be described as a comment